newm-server/src/main/kotlin/io/newm/server/features/minting/repo/MintingRepositoryImpl.kt

```diff
@@ -591,8 +591,26 @@ class MintingRepositoryImpl(
                     }
                     add(
                         plutusDataMapItem {
-                            mapItemKey = "copyright".toPlutusData()
-                            mapItemValue = song.copyright!!.toPlutusData()
+                            mapItemKey = "composition_copyright_owner".toPlutusData()
+                            mapItemValue = song.compositionCopyrightOwner!!.toPlutusData()
+                        }
+                    )
+                    add(
+                        plutusDataMapItem {
+                            mapItemKey = "composition_copyright_year".toPlutusData()
+                            mapItemValue = song.compositionCopyrightYear!!.toPlutusData()
+                        }
+                    )
+                    add(
+                        plutusDataMapItem {
+                            mapItemKey = "phonographic_copyright_owner".toPlutusData()
+                            mapItemValue = song.phonographicCopyrightOwner!!.toPlutusData()
+                        }
+                    )
+                    add(
+                        plutusDataMapItem {
+                            mapItemKey = "phonographic_copyright_year".toPlutusData()
+                            mapItemValue = song.phonographicCopyrightYear!!.toPlutusData()
```

Comment:
@AndrewWestberg verify that this is what we want to do here

Reply:
@wlara You should instead build the fields into a single string value.

`mapItemValue = "℗ ${song.phonographicCopyrightYear} ${song.phonographicCopyrightOwner}, © ${song.compositionCopyrightYear} ${song.compositionCopyrightOwner}".toPlutusData()`

Reply:
@AndrewWestberg Fixed